mathesar_db/tests/conftest.py

```diff
@@ -0,0 +1,52 @@
+"""
+This file should provide utilities for setting up test DBs and the like.  It's
+intended to be the containment zone for anything specific about the testing
+environment (e.g., the login info for the Postgres instance for testing)
+"""
+import pytest
+from sqlalchemy import create_engine, text
+from config.settings import DATABASES
+
+TEST_DB = "mathesar_db_test_database"
+
+
+@pytest.fixture(scope="session")
+def test_db():
+    superuser_engine = _get_superuser_engine()
+    with superuser_engine.connect() as conn:
+        conn.execute(text("COMMIT"))
```

Comment:
why use raw SQL here?

Reply:
For some reason, `conn.commit()` doesn't end an empty transaction.  The reason that's important is that you can't execute `DROP DATABASE` or `CREATE DATABASE` inside of a transaction block, but `engine.connect` always starts an implied transaction block in SQLAlchemy.  Admittedly, I didn't spend lots of time investigating _why_ `conn.commit()` doesn't seem to end the transaction block when it's empty, since it's just a test file.

Reply:
Could you add that as a comment? also maybe a TODO to investigate?